templates/related-accessories.php

```diff
@@ -12,7 +12,7 @@
   }
 } ?>
 
-<div class="related-accessories<?= $is_notice_template ? '' : '--summary' ?>">
+<div class="related-accessories<?= $is_notice_template ? '' : '--summary' ?>" <?= $is_hidden ? 'style="display: none"' : '' ?> >
```

Comment:
Ditto - We should be able to detect this in the frontend by looking for the special `button.ajax_add_to_cart` class on the page.